client/state/inline-help/selectors.js

```diff
@@ -33,7 +33,8 @@ export function getSelectedResult( state ) {
  * @return {Boolean}        Whether search results are being requested
  */
 export function isRequestingInlineHelpSearchResultsForQuery( state, searchQuery ) {
-	return !! get( state, 'inlineHelpSearchResults.requesting.' + searchQuery );
+	const allRequesting = get( state, 'inlineHelpSearchResults.requesting' );
+	return !! get( allRequesting, [ searchQuery ] );
```

Comment:
We could use `has` here, which would return a boolean to stat with.
Also, is there benefit to splitting out in to 2 calls? Maybe `has( state, [ 'inlineHelpSearchResults', 'requesting', searchQuery ] )` could work?

Reply:
Not a blocker btw, just a suggestion/thought :)

Reply:
Nice idea! `has` doesn't work here though since it can be both set and false at the same time. Using `has` that would always return `true`.

Reply:
Got ya! I thought that might be a possibility! Thanks for the info!